### ISSUE TITLE
Add FaceBook or Google login provider, how ?

### ISSUE THREAD

**Issue by vd3d** [abpframework/abp#2799]
Hi,

I wish a new login provider, google or facebook.

But how ? (and in which project, I mean xxxWeb, xxxApiHost, xxxIdentity)

I have add the following code, but it doesn't help !

```
.AddGoogle(googleOptions =>
                {
                    // Configure here : https://console.developers.google.com/
                    googleOptions.SignInScheme = IdentityServer4.IdentityServerConstants.ExternalCookieAuthenticationScheme;
                    googleOptions.ClientId = configuration["Authentication:Google:ClientId"];
                    googleOptions.ClientSecret = configuration["Authentication:Google:ClientSecret"];
                });
```

Thanks

**Comment by hikalkan** [comment]
See https://github.com/abpframework/abp/issues/2787#issuecomment-583873851 and https://github.com/abpframework/abp/issues/2533#issuecomment-570939959 for a general response on the customization of the current modules.

**Comment by vd3d** [comment]
Sorry, but I don't understand your answer and the relationship with the comment #2787 & #2533 !

I have a MyProject.IdentityServer, and then I have simply add my code in the ConfigureService.
Like here : http://koscielniak.me/post/2016/05/google-identity-provider-with-identityserver4/

I don't think that I should modify or overwrite any project/module, right ?

PS: Also I have see in the web page that you have add some code to handle the external providers, but I didn't find any exemple !
